solutions/cf-702C.mdx

```diff
@@ -354,6 +354,7 @@ public class Main {
 		int r = 0;
 		for (int i = 0; i < n; i++) {
 			int distance = Integer.MAX_VALUE;
+			//The Lower() and Higher() Methods of the TreeSet class returns a value of the Type Integer here and null if there is no such element.
```

Comment:
comment too long
max line length 80 chars
```java
/*
 * and format the comment something
 * along the lines of this
 */
```

Reply:
Is it okay if I just format the same comment in the format you've written above? Or do I have to rewrite it?

Reply:
no the content is fine
it's just the formatting